### ISSUE TITLE
Usage within a composable seems to stop useSchemaOrg() from working ... 

### ISSUE THREAD

**Issue by michealroberts** [unjs/unhead#263]
### Environment

```json
{
    "devDependencies": {
       "nuxt": "^3.8.0",
       "nuxt-schema-org": "^3.0.1",
    },
}
```

Running Node v18.12.1

### Reproduction

To reproduce, simply define any schema, e.g.,

```ts
useSchemaOrg([
  defineWebSite({
    name: 'My Awesome Website',
  }),
  defineWebPage(),
])
```

Inside of a composable, and call said composable within a Nuxt 3 page ...

e.g.,

```ts
const useMySuperCoolSchema = () => {
  useSchemaOrg([
    defineWebSite({
      name: 'My Awesome Website',
    }),
    defineWebPage(),
  ])
}
```

The resulting JSON+LD script tag will not render ...

### Describe the bug

The bug occurs when using any of the provide composables from the @unhead/schema package within a composable.

**Comment by harlan-zw** [comment]
Thanks for the issue, this is actually a [nuxt-schema-org](https://github.com/harlan-zw/nuxt-schema-org) issue. I've pushed up a fix in 3.1.3

**Comment by michealroberts** [comment]
Thanks @harlan-zw I'll upgrade and confirm.
